pcsx2/Recording/RecordingControls.cpp

```diff
@@ -110,13 +110,11 @@ void RecordingControls::TogglePause()
 void RecordingControls::Pause()
 {
 	fStop = true;
-	fFrameAdvance = true;
 }
 
 void RecordingControls::Unpause()
 {
 	fStop = false;
 	fStart = true;
-	fFrameAdvance = true;
 }
 #endif
```

Comment:
why these changes exactly?

Reply:
That was something from one of your commits. (https://github.com/PCSX2/pcsx2/pull/3669/commits/85560a4bf144ccd5bb1e75c9b3bcb2817a56f4af)

Reply:
🤦 thank god i described that commit.